src/main/resources/assets/admin/common/js/schema/content/ContentTypeSummary.ts

```diff
@@ -10,26 +10,29 @@ export class ContentTypeSummary
     extends Schema
     implements Equitable {
 
-    private allowChildContent: boolean;
+    private readonly allowChildContent: boolean;
 
-    private abstract: boolean;
+    private readonly abstract: boolean;
 
-    private final: boolean;
+    private readonly final: boolean;
 
-    private superType: ContentTypeName;
+    private readonly superType: ContentTypeName;
 
-    private displayNameExpression: string;
+    private readonly displayNameExpression: string;
 
-    private displayNameLabel: string;
+    private readonly displayNameLabel: string;
 
-    private modifier: string;
+    private readonly modifier: string;
 
-    private owner: string;
+    private readonly owner: string;
 
-    private metadata: MixinNames;
+    private readonly metadata: MixinNames;
+
+    private readonly allowChildContentType: string[];
```

Comment:
We should rename this to `allowedChildContentTypes` (I know it's called like that in API, but we don't have to follow incorrect naming in TS classes).